### DISCUSSION TITLE
Failed to execute 'setItem' on 'Storage': Setting the value of 'wagmi.cache' exceeded the quota

### DISCUSSION THREAD

**Discussion by minhhoang2404** [wevm/wagmi#2647] [Q&A]
hi guy, I use wagmi and react querry. Please help me fix error. Thank you.
![image](https://github.com/wagmi-dev/wagmi/assets/50880080/c5a82cad-ff48-4c1f-9806-071c46c2f374)

**Comment by minhhoang2404** [comment]
hi @jxom , please help me baby <3

**Reply by a-somani** [reply]
i have the same issue. 

wagmi.cache, Size: 4314.69 KB, Query Sizes: 0.322265625 KB, 0.326171875 KB, 0.322265625 KB, **_4310.1396484375 KB_**, 0.5712890625 KB, 0.4375 KB, 0.43359375 KB, 0.541015625 KB, 0.4375 KB, 0.43359375 KB, 0.626953125 KB 

Looking at localStorage, theres one giant query by wagmi where its storing a contract ABI.

**Comment by vinerz** [comment]
Having the same issue here. Did you ever manage to find the root cause?
